UnityProject/Assets/Scripts/Items/Food/Donut.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.Collections;
+using UnityEngine;
+
+
+public class Donut : Edible
```

Comment:
You could simplify this component into a Heel job edible, so in the editor you can specify what jobs it heals for and how much

Reply:
only thing missing is changing the name of the component so it is relevant for any other edible that heals a job.

Reply:
what do i name it to again?

Reply:
Rename the Class to something like HealJobEdible, and then you'll need to rename the file in unity to that as well (might need to replace the component on the donuts)

Reply:
ooohhh, gotcha. Will do that once Unity stops crashing.